index.html

```diff
@@ -9,5 +9,8 @@
     <body>
         <h1>Git Basics</h1>
         <h2>Git Basics - General description</h2>
+        <div>
+            <h3>New Feature</h3>
```

Comment:
Can you pls, change the h3 tag to label tag, since we are using that way across the project